src/compiler-debugging.md

```diff
@@ -185,6 +185,40 @@ stack backtrace:
 
 Cool, now I have a backtrace for the error!
 
+## Getting the the error creation location
+
+`-Z track-diagnostics` can help figure out where errors are emitted. It uses `#[track_caller]`
+for this and prints its location alongside the error:
+
+```bash
```

Comment:
I went with consistency with the treat-as-err example above, which also uses bash syntax to set env variables

Reply:
you may fix that as well if you like

Reply:
Done

Reply:
I would not separate commands and their outputs as you did, especially since it does not help with highlighting at all. I can make the changes for you, if you feel like this is too much back-n-forth.

Reply:
Sorry for the delay - I  somewhat forgot about this Pr. I think the current separation is fine. Feel free to change it if you disagree.